example-apps/demo/source/components/injector-tree/injector-tree-dummy-1.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  Component,
+  Inject,
+} from '@angular/core';
+
+import Service3 from '../../services/service3';
+
+@Component({
+  selector: 'injector-tree-dummy-1',
+  template: `
+    <p>injector-tree-dummy-1 init: service3</p>
+    {{service3Value}}
+    <hr/>
+    <injector-tree-dummy-3></injector-tree-dummy-3>
+    <injector-tree-dummy-4></injector-tree-dummy-4>
+  `
+})
+export default class InjectorTreeDummy1 {
```

Comment:
Honestly, it didn't cross my mind to change it, but since we've been consistent in Augury itself I'll change it here as well, thanks.